pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/PercentileRawTDigestAggregationFunction.java

```diff
@@ -0,0 +1,142 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.core.query.aggregation.function;
+
+import com.tdunning.math.stats.TDigest;
+import java.util.Map;
+import org.apache.pinot.common.request.context.ExpressionContext;
+import org.apache.pinot.common.utils.DataSchema.ColumnDataType;
+import org.apache.pinot.core.common.BlockValSet;
+import org.apache.pinot.core.query.aggregation.AggregationResultHolder;
+import org.apache.pinot.core.query.aggregation.groupby.GroupByResultHolder;
+import org.apache.pinot.segment.local.utils.CustomSerDeUtils;
+import org.apache.pinot.segment.spi.AggregationFunctionType;
+import org.apache.pinot.spi.utils.BytesUtils;
+
+
+/**
+ * The {@code PercentileRawTDigestAggregationFunction} returns the serialized {@code TDigest} data structure of the
+ * {@code PercentileEstAggregationFunction}.
+ */
+public class PercentileRawTDigestAggregationFunction extends BaseSingleInputAggregationFunction<TDigest, String> {
+  private final PercentileTDigestAggregationFunction _percentileRawTDigestAggregationFunction;
+
+  public PercentileRawTDigestAggregationFunction(ExpressionContext expressionContext, int percentile) {
+    this(expressionContext, new PercentileTDigestAggregationFunction(expressionContext, percentile));
+  }
+
+  public PercentileRawTDigestAggregationFunction(ExpressionContext expressionContext, double percentile) {
+    this(expressionContext, new PercentileTDigestAggregationFunction(expressionContext, percentile));
+  }
+
+  protected PercentileRawTDigestAggregationFunction(ExpressionContext expression,
+      PercentileTDigestAggregationFunction percentileRawTDigestAggregationFunction) {
```

Comment:
Rename the parameter
```suggestion
      PercentileTDigestAggregationFunction percentileTDigestAggregationFunction) {
```

Reply:
Updated.

Reply:
Still not correct